pkg/status/render/fixtures/check_stats.text

```diff
@@ -38,6 +40,7 @@ Collector
     file_handle
     -----------
       Instance ID: file_handle [ERROR]
+      Long Running Check: false
```

Comment:
Do you think it's important to show for regular checks? Can it be omitted?

Reply:
IMO we could omit it there

Reply:
Done in the last commit 👍